currencies.json

```diff
@@ -779,18 +779,6 @@
     "deprecated": true,
     "core": false
   },
-  {
```

Comment:
I see three options:

1. Ask Community and John if this approach is okay.

2. Work with the existing fields
```
"note": "Removed support in Oct 2024",
"ignoreUpdates": true
```

3. Extend the table (preferred):
```
"supportRemovedAt": "date",
"ignoreUpdates": true
```

Reply:
Thanks for the options. I will bring this in community.